FABulous/fabric_generator/fabric_gen.py

```diff
@@ -836,28 +839,20 @@ def generateTile(self, tile: Tile) -> None:
         self.writer.addPortScalar("UserCLKo", IO.OUTPUT, indentLevel=2)
 
         if self.fabric.configBitMode == ConfigBitMode.FRAME_BASED:
-            if tile.globalConfigBits > 0:
-                self.writer.addPortVector(
-                    "FrameData", IO.INPUT, "FrameBitsPerRow -1", indentLevel=2
-                )
-                self.writer.addComment("CONFIG_PORT", onNewLine=False, end="")
-                self.writer.addPortVector(
-                    "FrameData_O", IO.OUTPUT, "FrameBitsPerRow -1", indentLevel=2
-                )
-                self.writer.addPortVector(
-                    "FrameStrobe", IO.INPUT, "MaxFramesPerCol -1", indentLevel=2
-                )
-                self.writer.addComment("CONFIG_PORT", onNewLine=False, end="")
-                self.writer.addPortVector(
-                    "FrameStrobe_O", IO.OUTPUT, "MaxFramesPerCol -1", indentLevel=2
-                )
-            else:
-                self.writer.addPortVector(
-                    "FrameStrobe", IO.INPUT, "MaxFramesPerCol -1", indentLevel=2
-                )
-                self.writer.addPortVector(
-                    "FrameStrobe_O", IO.OUTPUT, "MaxFramesPerCol -1", indentLevel=2
-                )
+            self.writer.addPortVector(
+                "FrameData", IO.INPUT, "FrameBitsPerRow -1", indentLevel=2
+            )
+            self.writer.addComment("CONFIG_PORT", onNewLine=False, end="")
+            self.writer.addPortVector(
+                "FrameData_O", IO.OUTPUT, "FrameBitsPerRow -1", indentLevel=2
+            )
+            self.writer.addPortVector(
+                "FrameStrobe", IO.INPUT, "MaxFramesPerCol -1", indentLevel=2
+            )
+            self.writer.addComment("CONFIG_PORT", onNewLine=False, end="")
+            self.writer.addPortVector(
+                "FrameStrobe_O", IO.OUTPUT, "MaxFramesPerCol -1", indentLevel=2
+            )
```

Comment:
It's unrelated to the changes you did, but maybe we also remove the whitespace in all of these occurences?
```suggestion
            self.writer.addPortVector(
                "FrameData", IO.INPUT, "FrameBitsPerRow-1", indentLevel=2
            )
            self.writer.addComment("CONFIG_PORT", onNewLine=False, end="")
            self.writer.addPortVector(
                "FrameData_O", IO.OUTPUT, "FrameBitsPerRow-1", indentLevel=2
            )
            self.writer.addPortVector(
                "FrameStrobe", IO.INPUT, "MaxFramesPerCol-1", indentLevel=2
            )
            self.writer.addComment("CONFIG_PORT", onNewLine=False, end="")
            self.writer.addPortVector(
                "FrameStrobe_O", IO.OUTPUT, "MaxFramesPerCol-1", indentLevel=2
            )
```

Reply:
Done!